src/ci_opam.ml

```diff
@@ -274,9 +274,11 @@ with_fold "Reverse.dependencies" (fun () ->
       let revdep_count = List.length packages in
       echo "\nREVDEPS %d total" revdep_count;
       let packages = List.fold_left (fun acc pkg ->
-          match max_version pkg with
-          | Some pkgv -> pkgv::acc
-          | None -> echo "Skipping uninstallable REVDEP %s" pkg; acc
+	  if List.mem pkg pins then pkg::acc else begin
+            match max_version pkg with
+            | Some pkgv -> pkgv::acc
+	    | None -> echo "Skipping uninstallable REVDEP %s" pkg; acc
```

Comment:
there's a minor whitespace issue, `Some` and `None` are not aligned in lines 279/280

Reply:
yikes, thanks for the comment; changed my editor config and it turns out it's been putting tabs in everything 🙀